moonshot/api.py

```diff
@@ -126,13 +127,12 @@
     "api_get_all_result_name",
     "api_read_result",
     "api_read_results",
-    "api_create_session",
-    "api_delete_session",
-    "api_get_all_session_detail",
-    "api_get_all_session_name",
-    "api_get_session",
-    "api_get_session_chats_by_session_id",
-    "api_send_prompt",
+    "api_get_all_session_names",
+    "api_get_all_session_metadata",
+    "api_load_session",
+    "api_get_available_session_info",
     "api_update_context_strategy",
     "api_update_prompt_template",
+    "api_run_red_teaming",
```

Comment:
i thought red teaming should be an instance function?

Reply:
for running of bm and rt from the api, do we run them with a function directly? 
say a user wants to run bm from the cli, what will he call?

Reply:
that should be done from integrations. so in cli, user type run_redteam, then behind got some methods to do whatever you typed in the run_redteaming